google/cloud/pubsub/subscriber_connection.cc

```diff
@@ -121,8 +123,17 @@ std::shared_ptr<pubsub::SubscriberConnection> MakeSubscriberConnection(
     std::vector<std::shared_ptr<SubscriberStub>> stubs,
     std::unique_ptr<pubsub::RetryPolicy const> retry_policy,
     std::unique_ptr<pubsub::BackoffPolicy const> backoff_policy) {
+  auto default_retry_policy = [] {
+    // Subscribers are special, by default we want to retry essentially forever
```

Comment:
s/, by/. By/

Or s/, by/: by/ perhaps?

Reply:
Done (the second).